.github/pull_request_template.md

```diff
@@ -1,9 +1,16 @@
 ## Issue
 
+<!-- If you want to close a issue with your PR use magic words such as closes, fixes or resolves "issue number" to autmatically link a PR to an issue. For example: Closes #0000, Fix: #0000 or Resolves: #0000 -->
+
```

Comment:
Let's give fewer options for the non-github-savy users :D

```suggestion
<!-- If you want to close an issue automatically when your PR is merged, write "Closes X" where X is the PR number. For example: Closes #000 -->

```

Reply:
Fair enough 😅